src/atomic.py

```diff
@@ -77,6 +77,11 @@ def look_at_point(x_node, y_node, z_node):
 	evl.look_at_point(x, y, z)
 	return TruthValue(1, 1)
 
+def look_at_face_point(x, y, z):
+	evl.look_at_point(x, y, z)
+	evl.gaze_at_point(x, y, z)
```

Comment:
You don't need both look and gaze.  One is enough. Look will turn the neck, so that the entire head is pointing in that direction. Gaze moves only the eyes, and NOT the neck/head. 

Thus, if you use look, the gaze command does nothing new or additional.